test/integration-test-v2.py

```diff
@@ -29,7 +29,6 @@ def main():
 
     test_multidomain()
     test_wildcardmultidomain()
-    test_overlapping_wildcard()
```

Comment:
Could you comment this out and add a comment explaining why it’s disabled instead of just outright removing it.